certified-connectors/Charles Universe/apiDefinition.swagger.json

```diff
@@ -0,0 +1,232 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Charles Universe",
+    "description": "The Charles Universe connector allows to react to events your Universe installation is generating. \n\n That is a e.g. send a Teams or Slack message when a person is created.",
```

Comment:
@eljefedelrodeodeljefe The description can be improved a bit, Can you please make it free from grammatical mistakes. Ex.: `...This is an e.g. send...`